lib/sqlalchemy_ingres/base.py

```diff
@@ -221,14 +219,23 @@ def get_column_specification(self, column, **kwargs):
         if str(column.type) == "VARCHAR":
             if column.type.length is None:   # If no length is provided for VARCHAR column
                 column.type.length = 255     # Set length to 255
-        
-        colspec = (
-            self.preparer.format_column(column)
-            + " "
-            + self.dialect.type_compiler_instance.process(
-                column.type, type_expression=column
+
+        if (sqlalchemy_version_tuple >= (2,0)):
```

Comment:
super pendatic, this is not pep8 compliant. 

    if (sqlalchemy_version_tuple >= (2,0)):
versus
    if (sqlalchemy_version_tuple >= (2, 0)):

My recommendation is we [black](https://github.com/psf/black) the code base in a new PR sometime (soon 🙏 ).

Reply:
Added space to comply with PEP8. Was `(2,0)`, now `(2, 0)`.

Will improve code formatting (via black or equivalent) via [II-16370](https://actian.atlassian.net/browse/II-16370)